playbooks/roles/os_temps/tasks/login_to_cluster.yml

```diff
@@ -1,4 +1,4 @@
 ---
 # login to the cluster
 - name: "Login to the Openshift cluster"
-  shell: "{{ oc_bin }} login {{ cluster_ip }} --username={{ username }} --password={{ password }} --insecure-skip-tls-verify=true"
\ No newline at end of file
+  command: "{{ oc_bin }} login {{ cluster_ip|quote }} --username={{ username|quote }} --password={{ password|quote }} --insecure-skip-tls-verify=true"
```

Comment:
Will this login maintain within the shell if done this way for the next call?

Reply:
yes. its seems its stored in a file somewhere, so it does not even matter which binary you've used to login.
Also seems to work across `oc` versions - but that is probably not guaranteed.